tests/sqllogictests/suites/base/11_data_type/11_0006_data_type_decimal

```diff
@@ -137,3 +137,30 @@ query I
 SELECT ANY(CAST(2.34 AS DECIMAL(6, 2)))
 ----
 2.34
+
+## insert
+
+statement ok
+drop table if exists t_decimal_4_2;
+
+statement ok
+create table t_decimal_4_2(a Decimal(4,2));
+
+statement ok
+insert into t_decimal_4_2 values (1.1), (0.1), (1.0), (1.1e1), (1.1e-1);
```

Comment:
We need more edge tests for decimal, include negative.

Reply:
@BohuTANG  ok, currently more tests is in unit test (faster for debug/modify)

Reply:
logic test is high-level test.
For example, we can do it as:
1, what's the max decimal define we can do in Databend.
2, Give a defination, like the test we have what's the range of it.


If we have these, it's easy to write documents and understand the decimal type better.